docs/contributing/contributing.md

```diff
@@ -0,0 +1,115 @@
+
+## Reviewing technical content PRs 
+
+Some of the best content contributions come during the PR review cycles. Follow best practices for technical content PR reviews just like you do for code reviews. For in-line suggestions, use the [GitHub suggesting feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request). The PR owner can merge in your suggested commits one at a time or in batch (preferred). When you do a more granular extensive review that results in more than 20 in-line suggestions, go ahead and check out the branch and make the changes yourself. 
+
+## Writing and contributing
+
+We welcome contributions to the tutorials. 
+
+Our technical content follows the [Google developer documentation style guide](https://developers.google.com/style). Highlights to help you get started:
+
+- [Highlights](https://developers.google.com/style/highlights)
+- [Word list](https://developers.google.com/style/word-list)
+- [Style and tone](https://developers.google.com/style/tone)
+- [Writing for a global audience](https://developers.google.com/style/translation)
+- [Cross-references](https://developers.google.com/style/cross-references)
+- [Present tense](https://developers.google.com/style/tense)
+
+The Google guidelines include more material than is listed here and are used as a guide that enables easy decision making about proposed content changes. 
+
+Other useful resources:
+
+- [Google Technical Writing Courses](https://developers.google.com/tech-writing)
+- [GitHub Guides Mastering Markdown](https://guides.github.com/features/mastering-markdown/)
+
+## Where can I find the tutorials and docs?
+
+Technical content includes Knowledge Base article and interactive tutorials. 
+ 
+- The Knowledge Base content is in the `docs/kb` folder. 
+- The Starport Developer Guide content is in the `docs/guide` folder. 
+
+Locations and folders for other content can vary. Explore the self-describing folders for the content that you are interested in. Some articles and tutorials reside in a single Markdown file while sub-folders might be present for other tutorials.
+
+As always, work-in-progress content might be happening in other locations and repos.
+
+## Who works on the tutorials?
+
+- The Tendermint (All in Bits) DevX team develops the tutorial code and dependencies. Our team includes [@fadeev](https://github.com/fadeev), [@lubtd](https://github.com/lubtd), [@ilgooz](https://github.com/ilgooz), [@Pantani](https://github.com/Pantani), [@shapkarin](https://github.com/shapkarin), and other talented developers. Don't they do an amazing job?
```

Comment:
```suggestion
- The Tendermint (All in Bits) DevX team develops the tutorial code and dependencies. Our team includes [@fadeev](https://github.com/fadeev), [@lubtd](https://github.com/lubtd), [@ilgooz](https://github.com/ilgooz), [@Pantani](https://github.com/Pantani), and other talented developers. Don't they do an amazing job?
```

Let's not get ahead of ourselves 😉

Reply:
made a good change that a wise colleague suggested!